packages/account/src/Components/text-container/text-container.jsx

```diff
@@ -1,3 +1,7 @@
 import React from 'react';
 
-export const TextContainer = ({ children }) => <div className='account__text_container'>{children}</div>;
+export const TextContainer = ({ children }) => (
+    <div className='account__text_container' data-testid='test_text_container'>
+        {children}
```

Comment:
prefix test id's with dt.  refer the doc - https://wikijs.deriv.cloud/en/Frontend/General-information/test_guide